sources/api/migration/migrations/v0.3.2/migrate-admin-container/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "migrate-admin-container"
```

Comment:
Can we add the version to this?  The directory is scoped under v0.3.2, but the binary name isn't, so we can never name another binary "migrate-admin-container" after this... if we have to come up with a naming scheme, it seems like the intended version (0.5.0) would be a fine suffix.

Reply:
Is `migrate-admin-container-v050` ok? Can't have `.`'s in cargo package names.

Reply:
Maybe `migrate-admin-container-v0_5_0`?

Reply:
Hey @tjkirch, this was dumb, migration filenames can't have underscores there.